starport/services/network/campaign.go

```diff
@@ -100,29 +63,61 @@ func (n Network) CampaignUpdateTotalShares(campaignID uint64, totalShares campai
 	return nil
 }
 
-// CampaignUpdateTotalSupply updates the campaign total supply
-func (n Network) CampaignUpdateTotalSupply(campaignID uint64, totalSupply sdk.Coins) error {
-	n.ev.Send(events.New(events.StatusOngoing, fmt.Sprintf(
-		"Updating the campaign %d total supply to %s",
-		campaignID,
-		totalSupply.String(),
-	)))
+// CampaignEdit updates the campaign name or metadata
+func (n Network) CampaignEdit(
+	campaignID uint64,
+	name string,
+	metadata []byte,
+	totalShares campaigntypes.Shares,
+	totalSupply sdk.Coins,
+) error {
+	account := n.account.Address(networktypes.SPN)
+	msgs := make([]sdk.Msg, 0)
+	if name != "" || len(metadata) > 0 {
+		n.ev.Send(events.New(events.StatusOngoing, fmt.Sprintf(
```

Comment:
I also think that these events aren't manageable, it's unnecessary to print these logs for each property we updated. Instead let's please do what's described [here](https://github.com/tendermint/starport/pull/2099#pullrequestreview-891841210).